angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -170,6 +170,10 @@
                 <nimble-list-option *ngFor="let item of comboboxItems" [ngValue]="item">{{ item ? item.first : '' }}</nimble-list-option>
             </nimble-combobox>
         </div>
+        <div class="sub-container">
+            <div class="container-label">Rich Text Viewer</div>
+            <nimble-rich-text-viewer></nimble-rich-text-viewer>
```

Comment:
Once you add the markdown Input, could you give this some markdown content so that it's not completely empty?

Reply:
Yes, I will be adding the `markdown` string to this example client app once the viewer component is completed.